packages/common/src/configs.ts

```diff
@@ -92,6 +92,30 @@ export const envConfigs: Record<EnvironmentType, ProtocolConfig[]> = {
       },
       metaTx: undefined,
       lens: undefined
+    },
```

Comment:
why are we not deleting the goerli and mumbai configs?

Reply:
We keep them for now, so that we can validate the upgrade v2.3.0 => v2.4.0 process (this is what will happen on production).
Once the migration to v2.4.0 is completed on Prod, we will eventually keep only Amoy and Sepolia (for Amoy, we also need to wait for Biconomy to support it)